.github/workflows/project-time-in-area-test-analytics.yml

```
@@ -0,0 +1,382 @@
+
+jobs:
+  test-visualization-script:
+    name: Test Track Heatmap Viewer
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python 3.13
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)